test_crypto/test_decrypt.py

```diff
@@ -0,0 +1,39 @@
+from decrypt import decryptedVerification
+from hashlib import sha256
+
+
+def test_decryptedVerification():
+    assert decryptedVerification(b'*\x0f\\i\xd0\x1a>,\x8e\xbe'
+                                 , sha256(b'\x99\xfa\xca\x9c\xfd\x03?\xda\xfcv')
+                                 , b'\xb3\xf5\x96\xf5-\x19\x01\xf6r\xc8') \
+    == b'\x99\xfa\xca\x9c\xfd\x03?\xda\xfcv'
+
+
+def test_decryptedVerification_two():
```

Comment:
You can use pytest parametrization, not gonna force you to do it now but it is a better way to write different inputs for similar test cases